molecule-gradle-plugin/src/main/java/app/cash/molecule/gradle/MoleculeExtension.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.molecule.gradle
+
+import org.gradle.api.provider.Property
+
+interface MoleculeExtension {
+  /**
+   * The version of the JetBrains Compose compiler to use, or a Maven coordinate triple of
```

Comment:
Is "Maven coordinate triple" a thing?

Reply:
I've seen it referenced that way in a few places. It's meant to mean you need all three parts, whereas "coordinate" on its own may sometimes mean only the group and artifact IDs.

https://www.google.com/search?q=maven+coordinate+%22triple%22